### PR TITLE
Space mono normal font

### PR DESCRIPTION
**Pull request by mittalyashu**
* Add space mono regular font
* Add me as a contributor
* Alphabetize Fonts

### CODE REVIEW

File: components/style/Font.js [hunk 1]
```diff
@@ -121,6 +121,17 @@ export default () => (
         unicode-range: U + 0000-00ff, U + 0131, U + 0152-0153, U + 02bb-02bc, U + 02c6, U + 02da,
           U + 02dc, U + 2000-206f, U + 2074, U + 20ac, U + 2122, U + 2212, U + 2215;
       }
+      
+      /* latin */
+      @font-face {
+        font-family: 'Space Mono';
+        font-style: normal;
+        font-weight: 400;
```
Comment (mfix22) [inline components/style/Font.js]: Please add `font-display: swap;` following the other font faces.

Comment (mittalyashu) [reply components/style/Font.js]: Should I add `font-display: swap;` to every font faces?

Comment (mfix22) [reply components/style/Font.js]: Ohhhh whoops! Never mind (: that change is on another PR (#458)

**Comment by mfix22** [comment]
It hasn't been deployed yet.

**Comment by mixn** [comment]
@mfix22 I’ve started watching this repo so I get instant updates that I’ll try to add to carbon-now-cli right away. :) It should ideally always mirror the web UI. [Was a bit late with these](https://github.com/mixn/carbon-now-cli/commits?author=mixn&since=2018-08-10T22:00:00Z&until=2018-08-11T22:00:00Z), then I realized I should have just watched 👀 earlier. 🤷‍♂️

**Comment by mfix22** [comment]
@mixn yeah we try and batch multiple updates into a release to the site. So just because it is merged into master doesn't mean it is live. You can check https://github.com/dawnlabs/carbon/releases to see if a feature is live or not.

**Comment by mixn** [comment]
@mfix22 I know, that’s what I have been doing since I started watching. :) I remember reading somewhere that you deploy like once a week or so (don’t quote me on the timeframe, but I definitely know it’s every `x days`). So I know the UI isn’t representative of what is already included.

**Pull request merged**